include/itkImageToImageOfVectorsFilter.h

```diff
@@ -75,14 +75,6 @@ class ITK_TEMPLATE_EXPORT ImageToImageOfVectorsFilter
   using OutputImageType = Image<OutputPixelType, ImageDimension>;
   using RegionType = typename OutputImageType::RegionType;
 
-  using DataObjectPointerArraySizeType = typename Superclass::DataObjectPointerArraySizeType;
-
-  virtual void
```

Comment:
Does `using Superclass::SetNthInput;` work? If so, that would be a better solution.

Reply:
@dzenanz Yes, this solution works as well. I will update my commit to include `using Superclass::SetNthInput` instead of deprecating the function.